src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -34,7 +34,6 @@
 from odemis.acq.stream import Stream, EMStream, ARStream, \
     SpectrumStream, FluoStream, MultipleDetectorStream, util, executeAsyncTask, \
     CLStream
-from odemis.gui import conf
 from odemis.gui.util.raster import point_in_polygon
```

Comment:
Actually, that import is also a bit fishy: normally the .acq should not have any import from .gui . If we need this function, we probably could move gui.util.raster to util.raster.

Reply:
good point. I don't really see a reason why `raster.py` is in `gui.util` and it is a very useful function to have, so I will move `gui.util.raster` to `util.raster`

Reply:
should I do this in a separate PR?

Reply:
Yes, that's probably the best.